CHANGELOG.md

```diff
@@ -1,4 +1,88 @@
+* cloudrunv2: added `build_config` to `google_cloud_run_v2_service` ([#20979](https://github.com/hashicorp/terraform-provider-google/pull/20979))
+* compute: added `pdp_scope` field to `google_compute_public_advertised_prefix` resource ([#20972](https://github.com/hashicorp/terraform-provider-google/pull/20972))
+* compute: adding `labels` field to resource_compute_interconnect_attachment ([#20971](https://github.com/hashicorp/terraform-provider-google/pull/20971))
+* compute: fixed a issue where `custom_learned_route_priority` was accidentally set to 0 during updates in 'google_compute_router_peer' ([#20952](https://github.com/hashicorp/terraform-provider-google/pull/20952))
+* filestore: added support for `tags` field to `google_filestore_instance` resource ([#20955](https://github.com/hashicorp/terraform-provider-google/pull/20955))
+* gemini: promoted resources `gemini_code_repository_index`, `gemini_repository_group` to GA. ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
+* networksecurity: added `custom_mirroring_profile` and `custom_intercept_profile` fields to `google_network_security_security_profile` and `google_network_security_security_profile_group`  resources ([#20990](https://github.com/hashicorp/terraform-provider-google/pull/20990))
+* pubsub: added `enforce_in_transit` fields to `google_pubsub_topic` resource ([#20926](https://github.com/hashicorp/terraform-provider-google/pull/20926))
+* pubsub: added `ingestion_data_source_settings.azure_event_hubs` field to `google_pubsub_topic` resource ([#20922](https://github.com/hashicorp/terraform-provider-google/pull/20922))
+* redis: added `psc_service_attachments` field to `google_redis_cluster` resource, to enable use of the fine-grained resource `google_redis_cluster_user_created_connections` ([#20977](https://github.com/hashicorp/terraform-provider-google/pull/20977))
+
+BUG FIXES:
+* apigee: fixed `properties` field update on `google_apigee_environment` resource ([#20987](https://github.com/hashicorp/terraform-provider-google/pull/20987))
+* artifactregistry: fixed perma-diff in `google_artifact_registry_repository` ([#20989](https://github.com/hashicorp/terraform-provider-google/pull/20989))
+* compute: fixed failure when creating `google_compute_global_forwarding_rule` with labels targeting PSC endpoint ([#20986](https://github.com/hashicorp/terraform-provider-google/pull/20986))
+* container: fixed `additive_vpc_scope_dns_domain` being ignored in Autopilot cluster definition ([#20937](https://github.com/hashicorp/terraform-provider-google/pull/20937))
+* container: fixed propagation of `node_pool_defaults.node_config_defaults.insecure_kubelet_readonly_port_enabled` in node config. ([#20936](https://github.com/hashicorp/terraform-provider-google/pull/20936))
```

Comment:
What resource?

Reply:
Given node config is affected, it's both (and they're the only resources in the product. LMK how you'd prefer we display that.

Reply:
Eh, if it's both resources in container then this could make sense by itself